addon/core/schema.ts

```diff
@@ -250,57 +310,60 @@ export function renderInvisibleRdfa(
             .flatMap((subj) => getBacklinks(subj.value))
             .find((bl) => bl?.predicate === predicate);
           if (backlinkToImportedResource) {
-            propElements.push([
-              'span',
-              {
-                about: backlinkToImportedResource.subject.value,
-                property: predicate,
-                resource: object.value,
-              },
-            ]);
+            propElements.push(
+              namedNodeSpan(
+                backlinkToImportedResource.subject.value,
+                predicate,
+                object.value,
+              ),
+            );
           }
         }
         break;
       }
       case 'Literal': {
-        if (object.language?.length) {
-          propElements.push([
-            'span',
-            {
-              property: predicate,
-              content: object.value,
-              lang: object.language,
-            },
-            '',
-          ]);
-        } else if (object.datatype?.value?.length) {
-          propElements.push([
-            'span',
-            {
-              property: predicate,
-              content: object.value,
-              datatype: object.datatype.value,
-            },
-            '',
-          ]);
-        } else {
-          propElements.push([
-            'span',
-            {
-              property: predicate,
-              content: object.value,
-            },
-            '',
-          ]);
-        }
+        propElements.push(literalSpan(predicate, object));
         break;
       }
     }
   }
+  const externalTriples = nodeOrMark.attrs['externalTriples'] as FullTriple[];
+  if (externalTriples.length) {
+    const externalElements = [];
+    for (const fullTriple of nodeOrMark.attrs[
```

Comment:
nit: `for (const fullTriple of externalTriples) {`?

Reply:
fixed in [590ec1a](https://github.com/lblod/ember-rdfa-editor/pull/1229/commits/590ec1a8da9429738752371660a9693116faab9e)